swingbits/src/main/java/org/oxbow/swingbits/table/filter/DistinctColumnItem.java

```diff
@@ -46,7 +46,9 @@ public DistinctColumnItem( Object value, int row) {
     }
 
     public Object getValue() {
-        return value;
+    	if (this.value instanceof Boolean) return this.value.toString();

```

Comment:
Not sure this is good. We have to return actual value here. If you need to improve the rendering - it has to be done in `DistinctColumnItem` itself. Please update PR accordingly.